workspaces/src/network/sandbox.rs

```diff
@@ -206,3 +234,106 @@ impl Sandbox {
         Ok(())
     }
 }
+
+#[cfg(feature = "experimental")]
+impl Sandbox {
+    pub(crate) async fn changes_in_block(&self, block_reference: BlockReference) -> Result<()> {
```

Comment:
This is not the right place to implement RPC methods. RPC methods are not methods of "sandbox" they are methods of RPC client, see https://github.com/near/workspaces-rs/blob/cda1dc4e481e91bb17d95053b42d3b584cf2a95b/workspaces/src/rpc/client.rs, where other RPC methods are already implemented.

Reply:
Thanks, I'll move the implementations